api/net/Areas/Subscriber/Controllers/WorkOrderController.cs

```diff
@@ -147,7 +147,9 @@ public async Task<IActionResult> RequestTranscriptionAsync(long contentId)
                     NotificationId = notificationId,
                     ContentId = contentId,
                     RequestorId = workOrder.RequestorId,
-                    To = !String.IsNullOrWhiteSpace(user.PreferredEmail) ? user.PreferredEmail : user.Email,
+                    To = user != null && !user.IsVacationMode()
```

Comment:
Not needed. Vacation mode is just to turn off emails.

Reply:
I understood this part would send email using the email account of `To` property. So I checked the email account is not under VacationMode.